jupyterhub/jupyterhub/base/jupyterhub-singleuser-profiles-configmap.yaml

```diff
@@ -24,10 +24,10 @@ data:
               value: $(s3_endpoint_url)
           resources:
             requests:
-              memory: 1Gi
-              cpu: 500m
+              memory: 8Gi
+              cpu: 1
             limits:
-              memory: 2Gi
+              memory: 16Gi
```

Comment:
this will affect defaults on all jupyterhub deployment since you've put this in `base`
can you apply this to osc overlays via patch instead?

Reply:
I can do that.  I'm a bit confused, since this PR is only against the CL2 branch.